mgr/gccfilter

```diff
@@ -0,0 +1,451 @@
+#!/usr/bin/env perl
```

Comment:
gccfilter should not be a requirement for ROOT6.  It might be nice to have, but... definitely falls under the category of "tools" not "build system".  Should be located elsewhere.